src/root.component.tsx

```diff
@@ -1,14 +1,46 @@
 import React from "react";
-import { Route } from "react-router";
-import { BrowserRouter } from "react-router-dom";
+import { BrowserRouter, Route } from "react-router-dom";
 import openmrsRootDecorator from "@openmrs/react-root-decorator";
 import PatientChartSummary from "./summary/patient-chart-summary.component";
+import PatientBanner from "./summary/banner/patient-banner.component";
+import { getCurrentPatient } from "@openmrs/esm-api";
+import { createErrorHandler } from "@openmrs/esm-error-handling";
+import { LevelTwoRoutes } from "./summary/level-two-routes";
 
 function Root(props) {
+  const [currentPatient, setCurrentPatient] = React.useState(null);
+  const [showPatientSummary, setShowPatientSummary] = React.useState(true);
+
+  React.useEffect(() => {
+    const subscription = getCurrentPatient().subscribe(patient => {
+      setCurrentPatient(patient);
+      createErrorHandler();
+    });
+
+    return () => subscription.unsubscribe();
+  });
   return (
-    <BrowserRouter>
-      <Route to="/patient/:patientUuid/chart" component={PatientChartSummary} />
-    </BrowserRouter>
+    <main className="omrs-main-content">
+      <BrowserRouter basename={window["getOpenmrsSpaBase"]()}>
+        <Route path="/patient/:patientUuid/chart">
+          <PatientBanner
+            match={props.match}
+            patient={currentPatient}
+            showPatientSummary={setShowPatientSummary}
+          />
+        </Route>
+        <Route
+          path="/patient/:patientUuid/chart"
+          exact
+          component={PatientChartSummary}
+        />
+        {currentPatient && (
+          <Route path="/patient/:patientUuid/chart">
```

Comment:
This is incorrect usage of react-router. The `props.match` that you're passing to LevelTwoRoutes is the wrong one - it's the one for the parent component and not for the inner route. See [`<Route render={...} />`](https://reacttraining.com/react-router/web/api/Route/render-func) for the better way to do this

```tsx
<Route path="/patient/:patientUuid/chart" render={routeProps => (
  <LevelTwoRoutes {...routeProps} patient={currentPatient} />
)} />
```